src/coreComponents/finiteVolume/FluxApproximationBase.hpp

```diff
@@ -255,6 +265,12 @@ class FluxApproximationBase : public dataRepository::Group
   /// length scale of the mesh body
   real64 m_lengthScale;
 
+  /// flag indicating whether C1-PPU scheme should be used
+  integer m_useC1PPU;
```

Comment:
Following what we discussed: in this PR or in subsequent ones, we could have here:
```
UpwindingScheme m_upwindingScheme;
```
or a `struct` containing this information and possibly more.

If you want an example of how to define an enum in GEOS, you can do the following:

1. You define the `enum class UpwindingScheme : integer` in `FluxApproximationBase.hpp` as done [here](https://github.com/GEOS-DEV/GEOS/blob/beec4f5b4d2cde2ff5e308f74200443c646a804f/src/coreComponents/physicsSolvers/NonlinearSolverParameters.hpp#L123)
2. You define the `ENUM_STRINGS` macro in the same file as done [here](https://github.com/GEOS-DEV/GEOS/blob/beec4f5b4d2cde2ff5e308f74200443c646a804f/src/coreComponents/physicsSolvers/NonlinearSolverParameters.hpp#L286)
3. You register them with automatic input checking in `FluxApproximationBase.cpp` as done [here](https://github.com/GEOS-DEV/GEOS/blob/beec4f5b4d2cde2ff5e308f74200443c646a804f/src/coreComponents/physicsSolvers/NonlinearSolverParameters.cpp#L135)

Reply:
in what namespace/class is better to place that UpwindingScheme?

Reply:
I think I addressed it, please have a look and let me know.